setup.py

```diff
@@ -1,18 +1,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from setuptools import setup
+from acis_taco.version import version
 
-from Chandra.taco.version import version
-
-setup(name='Chandra.taco',
+setup(name='acis_taco',
```

Comment:
While you're in here can you set it up for pytest? 
 Basically just copy/paste the example here: https://occweb.cfa.harvard.edu/twiki/bin/view/Aspect/SkaPython#Package_layout